src/BaseServiceProvider.php

```diff
@@ -111,6 +113,11 @@ public function registerAdminMiddleware(Router $router)
         }
     }
 
+    public function registerCustomAuthGuard()
+    {
+        View::composer('backpack::*', \Backpack\Base\app\Http\ViewComposers\AuthComposer::class);
```

Comment:
A more _Laravely_ way would be to have both a Façade **and** a helper and then to have arguments on slack/gitter about which is best :(